corehq/apps/app_manager/templates/app_manager/spec/mocha.html

```diff
@@ -3,8 +3,14 @@
 
 {% block dependencies %}
 <script src="{% static 'app_manager/js/form_workflow.js' %}"></script>
+<script src="{% static 'app_manager/js/download_async_modal.js' %}"></script>
 {% endblock %}
 
 {% block mocha_tests %}
 <script src="{% static 'app_manager/spec/form_workflow_spec.js' %}"></script>
+<script src="{% static 'app_manager/spec/download_async_modal_spec.js' %}"></script>
```

Comment:
hmm i always thought that was the BDD naming for test. don't remember where i picked it up, but i'm pretty sure it's a common thing. also in this case i like cause it's easier to find js tests vs python tests if you use a fuzzy file finder